lib/ubi_cli.rb

```diff
@@ -4,6 +4,7 @@ class UbiCli
   force_autoload = Config.production? || ENV["FORCE_AUTOLOAD"] == "1"
 
   FRAGMENTS = {
+    "fw" => "firewall",
```

Comment:
It might be worth considering re-using these fragments for the `...-fields` params as well. If these are consistently shortened in the commands anyway, it seems like using the same thing in the params makes sense.

This might also address the `firewall-rule-fields` issue as it could become something like `fwr-fields` presumably.

Reply:
To be explicit/concrete, this would mean changing things like `firewall-fields` as a param to `fw-fields`.

I'm a bit on the fence about it, to be honest, but seems worth considering.

Reply:
I'm on the fence about it as well.  We can change this easily later if we need to.